src/pages/help-centre/index.tsx

```diff
@@ -0,0 +1,353 @@
+    border-bottom: 1px solid var(--color-grey-8);
+
+    @media ${device.tabletL} {
+        padding: 8rem 0 4rem;
+    }
+`
+const StyledContainer = styled.div<StyledProps>`
+    @media ${device.tabletL} {
+        padding: 2rem 0 2rem 0;
+    }
+`
+const SearchSection = styled.section<StyledProps>`
+    ${Backdrop} {
+        max-height: ${(props) => (props.show ? '100rem' : '0')};
+        transition: ${(props) => (props.has_transition ? 'max-height 0.6s ease-in-out' : 'none')};
+        overflow: hidden;
+    }
+`
+
+const SearchCrossIcon = styled.img`
+    width: 2.3rem;
+    height: 2.3rem;
+    position: absolute;
+    top: 1.4rem;
+    right: 2rem;
+
+    :hover {
+        cursor: pointer;
+    }
+`
+
+const SearchForm = styled.form`
+    position: relative;
+    padding-left: 6.4rem;
+    border: 1px solid var(--color-grey-17);
+    border-radius: 4px;
+    width: 99.6rem;
+    height: 6.4rem;
+    max-width: 99.6rem;
+
+    @media ${device.Laptop} {
+        width: 100%;
+
+        svg {
+            width: 2.5rem;
+            height: 3.55rem;
+        }
+    }
+`
+const SearchIconBig = styled.img`
+    width: 2.3rem;
+    height: 2.3rem;
+    position: absolute;
+    left: 2.4rem;
+    top: 2rem;
+`
+const Search = styled.input`
+    width: 95%;
+    font-size: var(--text-size-m);
+    font-weight: 500;
+    color: var(--color-black);
+    background-color: var(--color-white);
+    border: none;
+    outline: none;
+    height: 6rem;
+
+    ::placeholder {
+        color: var(--color-grey-17);
+    }
+`
+
+const ResultWrapper = styled.div`
+    > :first-child {
+        margin-top: 4rem;
+        margin-bottom: 3.6rem;
+    }
+`
+
+const ArticleSection = styled.section`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding: 0 0 8rem;
+
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        padding: 0 0 8rem;
+    }
+`
+
+const RowDiv = styled.div<StyledProps>`
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: ${(props) => props.wrap};
+
+    @media ${device.tabletS} {
+        flex-direction: column;
+    }
+`
+
+const ResponsiveHeader = styled(Header)`
+    @media ${device.tabletL} {
+        text-align: center;
+    }
+    @media ${device.mobileL} {
+        font-size: 4rem;
+    }
+`
+
+// Since useContext can only be used in functional components
+// Wrap HelpCenter class component in a function plug in the context
+const HelpCenter = () => {
+    const is_eu_country = React.useContext(DerivStore)
+    return <HelpCentreClass is_eu_country={is_eu_country} />
+}
+
+class HelpCentreClass extends React.Component<HelpCenterProps, HelpCenterState> {
+    constructor(props) {
+        super(props)
+        this.state = {
+            all_articles: [],
+            all_categories: {},
+            search: '',
+            search_has_transition: false,
+            toggle_search: true,
+        }
+    }
```

Comment:
Can you refactor the page into functional component?

Reply:
Finally did it